src/wp-includes/media.php

```diff
@@ -992,23 +991,22 @@ function wp_get_attachment_image_srcset( $attachment_id, $size = 'medium' ) {
  *
  * @since 4.4.0
  *
- * @param int    $attachment_id Image attachment ID.
- * @param string $size          Optional. Name of image size. Default value: 'medium'.
- * @param array  $args {
- *     Optional. Arguments to retrieve attachments.
- *
- *     @type array|string $sizes An array or string containing of size information.
- *     @type int          $width A single width value used in the default `sizes` string.
- * }
+ * @param int          $attachment_id Image attachment ID.
+ * @param array|string $size          Size of image, either array or string.
```

Comment:
This should be: "Size of image. Image size or array of width and height values (in that order). Default 'medium'."
See https://core.trac.wordpress.org/ticket/34257